packages/jest-runtime/src/helpers.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import chalk from 'chalk';
+
+export const enhanceUnexpectedTokenMessage = (e: Error) => {
+  e.stack =
+    `${chalk.bold.red('Jest encountered an unexpected token')}
+
+This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not JavaScript.
```

Comment:
Would it be more accurate to say this you are trying to import a file which _node_ cannot parse?